media-gfx/librecad/files/librecad-2.1.3-CVE-2021-45341.patch

```diff
@@ -0,0 +1,53 @@
+From f3502963eaf379a429bc9da73c1224c5db649997 Mon Sep 17 00:00:00 2001
```

Comment:
Please add a link to where you got it from and also any relevant Gentoo bugs.

Reply:
fixed